src/detail-panels/labeled/index.jsx

```diff
@@ -96,6 +97,7 @@ export default function LabeledPanel() {
   const { data: labelers, isLoading: isLoadingLabelers } = useLabelers();
 
   const { data: labels, isLoading } = useLabeled(did, labelers);
+  const labelsCount = useFeatureFlag('labels-count')
```

Comment:
same here

Reply:
this this is one last api call that wasn't addressed in the last round of changes